app/gui/tests/language_server.rs

```diff
@@ -64,7 +64,7 @@ wasm_bindgen_test_configure!(run_in_browser);
 async fn ls_text_protocol_test() {
     let _guard = enso_gui::initializer::setup_global_executor();
     let ide = setup_ide().await;
-    let project = ide.current_project();
+    let project = ide.current_project().expect("IDE is configured without an open project");
```

Comment:
We try to keep errors messages as nicely formatted sentences, ending with a dot.

Reply:
🙈 Always forget about that